### DISCUSSION TITLE
Add in Theme Playback Panel

### DISCUSSION THREAD

**Discussion by Roxo7777** [z-huang/music#1406] [Ideas]
The part I drew in red is the background, and in the Muzio Player player (used as an example), we have the playback background with the main colors of the album in acrylic, in YouTube Music we have it with the colors in gradient hex, and in InnerTune, it could be in Material You folder.

(if there was an inaccuracy, it was probably due to Google Translate)

https://drive.google.com/file/d/1Nmswbhlt3FrQ7DRAlVxXPLjfZfMu8JsS/view?usp=drivesdk

**Comment by z-huang** [comment]
Cannot access the link.

**Reply by Roxo7777** [reply]
I forgot the privacy settings, now you can see
